GE-Ontology/ontology/CPS.sadl

```diff
@@ -12,7 +12,7 @@
 
 uri "http://arcos.turnstile/CPS" alias CPS.
```

Comment:
I thought this was confusing too. The overlay is defined (in the readme) to be turnstile specific. My idea was that the directory should have been called Turnstile-Overlay as the content isn't really GE specific.

Reply:
I'm going to leave these URIs like they are because we've been going with the following format for the overlays: 
`http://arcos.<program_name>/<company_name>`